src/index.html

```diff
@@ -6,9 +6,40 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav">
+      <div class="nav__logo">
+        <a class="nav__logo-link" href="#">
+          <img
+          class="nav__logo-img"
+          src="../src/images/logo.png"
+          alt="logo"
+          >
+        </a>
+      </div>
+      <nav>
+        <ul class="nav__links-list">
+          <li class="nav__links-item "><a class="nav__links-link is-active" href="#">Apple</a></li>
+          <li class="nav__links-item"><a class="nav__links-link" href="#">Samsung</a></li>
```

Comment:
checklist p2 - use the names from that example, it would be more readable

and use meanfull hrefs (#apple)